src/Lint.jl

```
@@ -245,4 +245,28 @@ function lintexpr( ex::Any, ctx::LintContext )
     end
 end
 
+function lintserver(port)
+    server = listen(port)
+    while true
+      conn = accept(server)
+      @async begin
+        try
+          while true
+            line = readline(conn)
+            println(typeof(line))
+            println(ispath(strip(line)))
+            println(strip(line))
```

It would be probably better to throw an error, true.